templates/experiments/experiment_route_actions.html

```diff
@@ -0,0 +1,9 @@
+{% if record %}
+    {% url url_name request.team.slug record.parent_id record.id as action_url %}
```

Comment:
This feels hacky, but these templates are here because the URL scheme is different to the "generic" urls (it wants the parent experiment too).

Reply:
The only alternative I can think of is to use a function to get the URL instead of doing it in the template. That way you could pass a custom function as a parameter instead of overriding the template:

```
def get_action_url(request, record):
   ....
```